securitycenter/assets/main.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains examples calls to Cloud Security Center ListAssets API method.
+
+package main
+
+import (
+	securitycenter "cloud.google.com/go/securitycenter/apiv1beta1"
+	"context"
+	"fmt"
+	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/api/iterator"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1beta1"
+	"os"
+	"time"
+)
+
+// Lists all assets currently in an organization.
+// [START list_all_assets]
+func ListAllAssets(orgId string) (int, error) {
+	// Instantiate a context and a security service client to make API calls.
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		fmt.Printf("Error instantiating client %v\n", err)
+		return 0, err
```

Comment:
Idiomatic to do something like:
```
return 0, fmt.Errorf("securitycenter.NewClient: %v", err)
```

This should not print info about the error - the caller would do that. But, it's good to use `fmt.Errorf` to add additional info/context about the error being returned, like the name of the function that failed.

Same below with the other print + error calls.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings is somewhat relevant here.